graphql2/schema.graphql

```diff
@@ -503,7 +503,7 @@ input ScheduleTargetInput {
 }
 
 input ScheduleRuleInput {
-  id: ID @deprecated(reason: "ignored")
+  id: ID
```

Comment:
Are we un-deprecating this or should we instead now remove the field?

Reply:
It's not currently merged into the official spec (https://github.com/graphql/graphql-spec/pull/805) and gets flagged as invalid with the latest gqlparser where they added validation https://github.com/99designs/gqlgen/issues/1636

Since it caused the build to fail, currently not allowed, and the field is ignored (rather than having some legacy behavior) I just removed it from the schema. Open to alternatives, but it seemed very low risk